core/kernel/Float_spec.rb

```diff
@@ -198,6 +198,53 @@ def to_f() 1.2 end
     end
   end
 
```

Comment:
Maybe `"for hexadecimal literals with binary exponent"` reusing the wording in this PR :smile: